release_notes/ocp-4-14-release-notes.adoc

```diff
@@ -2986,6 +2986,46 @@ The default `IngressController` CR is configured with 4 HAProxy threads. If you
 
 * For {sno-caps} on {product-version} and Google Cloud Platform (GCP), there is a known issue with the Cloud Network Config Controller (CNCC) entering a `CrashLoopBackOff` state. This occurs at initialization time when the CNCC tries to reach the GCP internal load balancer address and the resulting hairpin traffic is not correctly prevented in OVN-Kubernetes shared gateway mode on GCP causing it to get dropped. Cluster Network Operator will show a `Progressing=true` status in such case. Currently, there is no workaround for this issue. (link:https://issues.redhat.com/browse/OCPBUGS-20554[*OCPBUGS-20554*])
 
+* Large latency spikes may be seen whenever a container is started, on an {sno-caps} with guaranteed cpus and load balancing disabled. (link:https://issues.redhat.com/browse/OCPBUGS-22901[*OCPBUGS-22901*])
```

Comment:
Would be good to specify what load balancing we refer to.
I.e. what is it that is load balanced or not

Reply:
Good point it is irq-load-balancing.crio.io to be honest there is an open question on whether I can report as noted as Red Hat internal

Reply:
@djuran changed to _with guaranteed CPUs and Interrupt Request (IRQ) load balancing is disabled_